lld/test/ELF/ppc32-debug-names.s

```diff
@@ -0,0 +1,417 @@
+# ppc-debug-names.s was generated with:
```

Comment:
match this with the file name (`ppc32-debug-names.s`)

Reply:
Done